api/src/middleware/publications.js

```diff
@@ -53,6 +53,21 @@ const createPublicationValidation = [
       .if(body("yearPublished")
       .exists())
       .isLength(4),
+    body("category.type", "Error: Category type must not be empty.")
+      .notEmpty(),
+    body("category.type", "Error: Category type does not match any of [\"CONFERENCE\", \"JOURNAL\"].")
+      .if(body("category.type")
+      .exists()
+      .notEmpty())
+      .isIn(["CONFERENCE", "JOURNAL"]),
+    body("category.categoryTitle", "Error: Category title must not be empty.")
+      .notEmpty(),
+    body("category.categoryTitle", "Error: Category title must be at least 3 characters.")
+    .if(body("category.categoryTitle")
+    .exists()
+    .notEmpty())
```

Comment:
Same as above.

Reply:
Same here. The first one is checking whether the title is empty and the second one is checking whether the title is at least has 3 characters given it is provided.